docs/sections/user_guide/installation.rst

```diff
@@ -1,42 +1,71 @@
+The recommended installation mechanism uses the Python package and virtual-environment manager :conda:`conda<>`. Specifically, these instructions assume use of the :miniforge:`Miniforge<>` variant of :miniconda:`Miniconda<>`, built to use, by default, packages from the :conda-forge:`conda-forge<>` project. Users of the original :miniconda:`Miniconda<>` or the :anaconda:`Anaconda distribution<>` should add the flags ``-c conda-forge --override-channels`` to ``conda`` commands to specify the required package channels.
 
-Users of the original Miniconda (or the :anaconda:`Anaconda distribution<>`) may need to add the flags ``-c conda-forge --override-channels`` to ``conda build``, ``conda create``, and ``conda install`` commands to specify the use of conda-forge packages.
+Use an Existing conda Installation
+----------------------------------
 
-Using a Fresh Miniforge Installation
-------------------------------------
+Install Into an Existing Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To install ``uwtools`` into an exiting environment in an existing conda (e.g. :miniforge:`Miniforge<>`, :miniconda:`Miniconda<>`, :anaconda:`Anaconda<>`) installation:
+
+#. Activate that environment.
+#. Identify the ``uwtools`` version number to install from the available versions shown by ``conda search -c ufs-community --override-channels uwtools``.
+#. Install ``uwtools`` into the active environment:
+
+  .. code-block:: text
+
+    conda install -c ufs-community uwtools=<version>
+
+Create a Standalone ``uwtools`` Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To create a standalone conda environment providing ``uwtools``:
+
+#. Identify the ``uwtools`` version number to install from the available versions shown by ``conda search -c ufs-community --override-channels uwtools``.
+#. Create the environment (here named ``uwtools`` via the ``-n`` flag, but any name may be used):
+
+  .. code-block:: text
+
+    conda create -n uwtools -c ufs-community uwtools=<version>
+
+Use a Fresh Miniforge Installation
+----------------------------------
+
+If no existing conda installation is available, install :miniforge:`Miniforge<>`
 
 .. include:: ../../shared/miniforge3_instructions.rst
 
-2. Install the ``conda-build`` and ``conda-verify`` packages into the base environment. If ``conda-build`` and ``conda-verify`` are already installed in your installation’s base environment, you may skip this step.
+2. Continue with the `Use an Existing conda Installation`_ instructions.
+
+Build the ``uwtools`` Package Locally
+-------------------------------------
+
+1. Install the necessary build packages into the conda installation's base environment (see the `Use a Fresh Miniforge Installation`_ instructions if an installation is unavailable or not writable):
```

Comment:
Does it make sense to use the `#` notation for numbered lists here since the numbers changed?

Reply:
The auto-numbering `#.` syntax seems to be thrown off by various things, for example `code-block` entries between items. We already had this problem with the `include`d Miniforge installation snippet, which provides item `1.` -- Sphinx did not automatically number the next bullet at `2.`, but restarted from `1.` (same thing happens with interspersed code blocks). Maybe there's a workaround for this, but I haven't found it yet. I like `#.`, but it seems to work as expected only in simple/limited cases.

Reply:
I'll add that I've had the same problem in the past with Markdown numbered lists, and I get it: It's not necessarily possible to know when a list is continuing vs when a new list is being started.